modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/clientExtensionFilters.spec.ts

```diff
@@ -163,3 +171,30 @@ test.describe('Client Extension Filters in Data Set Manager', () => {
 		});
 	});
 });
+
+modalFieldTest(
```

Comment:
I've included a simple test for each type of filter to avoid too much duplication.